### PR TITLE
Youtube Clone Pull Request

### PR DESCRIPTION
**Pull request by NataJenkins**
Youtube Clone page made with HTML and CSS.

In this project we implemented:
-Search bar with an tag
-A embeded video
-Buttons
-Sidebar
-Icons

This Branch is a copy of 'navbar" branch, thats why ther are not that much commits. we werent sure if delete the "navbar" branch or keep it, so we kept it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet"> 
+    <script src="https://kit.fontawesome.com/7bc3c221d9.js" crossorigin="anonymous"></script>
+    <title>Practice</title>
+</head>
+<body>
+                    <!--NAVBAR-->
+<div class="navbar">
```
Comment (kevcap) [inline index.html]: Use semantic HTML: here you should look for something more semantic than a `<div>`, you can check [this](https://www.w3schools.com/html/html5_semantic_elements.asp) link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    <div class="initial">
+    <img src="images/line.png" alt="line" >
+    <img src="images/YouTube.png" alt="line">
+</div>
+<div class="search">
+        <input type="text"  placeholder="Search.." > 
+    <div class="searchbtn">
```
Comment (kevcap) [inline index.html]: Here should be a `<button>` tag. 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+<div class="end">
+    <img src="images/camera.png" alt="line">
+    <img src="images/grid.png" alt="line">
+    <img src="images/bell.png" alt="line">
+</div>
+</div>
+
+                    <!--VIDEO PLAYER-->
+<div class="container">
```
Comment (kevcap) [inline index.html]: Use semantic HTML: here you should look for something more semantic than a `<div>`, you can check [this](https://www.w3schools.com/html/html5_semantic_elements.asp) link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+                <div class="button">
+                    <button type="button" id="cancel">CANCEL</button>
+                    <button type="button" id="comment">COMMENT</button>
+                </div>
+            </div>
+        </div>
+   
+
+                            <!--SIDEBAR-->
+<div class="sidebar">
```
Comment (kevcap) [inline index.html]: Use semantic HTML: here you should look for something more semantic than a `<div>`, you can check [this](https://www.w3schools.com/html/html5_semantic_elements.asp) link.

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Hi once more team, you did a very good job so far and your project is one of the best Youtube clones that I've reviewed until now, I`ll keep my eyes 👀 on you both, so congrats! 👏👏🥇



#### I can see the hard work that you put at this project, and we are almost done, there are just some required changes* that you have to implement in order to proceed:

_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
 
[REQUIRED CHANGES]
- [x] Please check the usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)  (You should have elements like `<main>`,`<article>`,`<section>`,`<aside>` and so on) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright) e.g.: the `<div>` tag is not semantic, so whenever possible, replace it for something that is related of what you're working at. e.g. instead of a `<div class="sidebar">` use the semantic tag `<aside class="sidebar">`, and so on.
- [x] Your code is throwing some errors and warnings at the [HTML5 Validator](https://validator.w3.org/nu/#textarear), please fix them.
image
- [x] Your navbar should have a `<button>` tag that represents the search button. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from pathwright)
- [x] The search bar should be aligned to the middle of the navbar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/searchbar_align_requirement.png) from Pathwright)
Tip: To do so, go to your .end class and remove that `margin-left: 20%` and go to your class .search and add `margin: 0 auto;` instead of `margin-left: auto`
- [x] Remove all the unnecessary repeated CSS code ([Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) Tip: Create one only class like this: .d-inline { display: inline-block } and use that same class when you need it.
image
- [x] Don't use `id` for styling, use class instead. ([Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/))



[OPTIONAL CHANGES]
- [x] Put your `css` files in a css folder. (To make your repo more professional)
- [ ] Improve the layout to make it more similar to the original youtube page;
- [ ] Improve the icons to make it more similar to the original youtube page;
- [ ] Add a favicon to make it more similar to the original youtube page;
- [ ] use `border-box` to add that bar on the top of the elements, instead of using `<hr>`
- [ ] Add the title of the video to the title of the page;
That recipe looks so delicious that now I'm hungry 😂😁

tip: you can use `!important`, but you shouldn't **overuse** it. I know that isn't the case, but I advise you to _prefer_ not using it at all while building these projects.

**Please make the required changes and resubmit for a new code review once you are ready**

## As always, I wish you the very best and see you both at the top🚀

  

<hr>

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Hi team, guess what? It's me again! Good job on making the previously required changes!
Your page meets most of the requirements, again, this one was close 😅  👏 
Team, you just have to implement some _minor_ changes in order to move on
 
## [REQUIRED CHANGES]
- [ ] Your code is still throwing some warnings at the [HTML5 Validator](https://validator.w3.org/nu/#textarear), should be easy to fix that!💪
image

## [OPTIONAL CHANGES]
Actually is mandatory, but since It wasn't pointed out earlier (by me btw), I'll mark it as optional:
- [x]  The share icon should be a `<button>` tag, not a `<a>` tag ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
image

## Please make the required changes and resubmit for a new code review once you are ready💪

You are this close 🤏 to finishing this project

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Surely is not Kevin again! Thanks for making that share button change, that was awesome!
Unfortunately, I still can see some errors in the HTML5 validator... 😕

Tip: Copy all your HTML code that is inside the [index.html](https://github.com/NataJenkins/Project-Embedding-Images-and-Video/blob/youTubePageCopy/index.html) and paste [here](https://validator.w3.org/#validate_by_input), so you'll see all the errors and will easily fix them 🔧
 
## [REQUIRED CHANGES]
- [ ] Your code is still throwing some warnings at the [HTML5 Validator](https://validator.w3.org/#validate_by_input), should be easy to fix that!💪
image


## Please make the required changes and resubmit for a new code review once you are ready💪

Close again, I'm sure that this will be the last time 😅

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by SpaYco — APPROVED** [review]
## Project Approved ✔️ 

great job making the changes requested by the TSE! 

you can merge and keep on learning with the next projects! 😄 

Best of luck, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**